pkg/docker/docker.go

```diff
@@ -347,16 +347,26 @@ func (d *stiDocker) GetOnBuild(name string) ([]string, error) {
 // and returns the image metadata
 func (d *stiDocker) CheckAndPullImage(name string) (*docker.Image, error) {
 	name = getImageName(name)
+	displayName := name
+
+	if glog.Is(3) == false {
```

Comment:
Why we don't need this on level 3? Looks like you've tried to write condition like `if logLevel < 3`. If it's true I see that you missed levels 4 and 5.